app/test_split_integer.py

```diff
@@ -2,20 +2,26 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(32, 6)) == 32), \
+        "Sum of the parts should be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert len(set(split_integer(16, 4))) == 1, \
+        "When value divisible by parts split into equal parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8, 1)[0] == 8, \
+        "Return part equals to value when split into one part"
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert (split_integer(16, 4) == sorted(split_integer(16, 4))), \
+        "Parts should be sorted when they are not equal"
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    diff = 10 - 6
+    assert split_integer(6, 10)[:diff] == [0] * diff, \
```

Comment:
Can you check it in a more simple way? Would be easier to compare certain input data and correct return data